output.go

```diff
@@ -146,6 +146,11 @@ func (o *Output) flushMetrics() {
 	sampleContainers := o.GetBufferedSamples()
 	start := time.Now()
 
+	o.logger.Debug("flushMetrics: Collecting...")
```

Comment:
Nit: remove the redundant logging. The following entry provides additional context with the count, so this log is superfluous.

Extra credit: a few lines down, we have the same situation from a previous contributor. Removal of the `o.logger.Debug("flushMetrics: Committing...")` would be appreciated.